src/components/homeTabs/GIVbacks.tsx

```diff
@@ -122,9 +122,13 @@ export const TabGIVbacksBottom = () => {
 			_rounStartTime.setDate(
 				tokenDistroHelper.startTime.getDate() + (_round - 1) * 15,
```

Comment:
The multiplier should be 14 not 15
```suggestion
				tokenDistroHelper.startTime.getDate() + (_round - 1) * 14,
```